android/src/main/java/com/amplitude/android/migration/IdentityStorageMigration.kt

```diff
@@ -18,7 +18,9 @@ class IdentityStorageMigration(
             if (identity.deviceId != null) {
                 destination.saveDeviceId(identity.deviceId)
             }
-            source.delete()
+            // Since we are not doing a major version upgrade, keep the old profile file around
+            // just in case someone decides to rollback back to the older version of the SDK.
+//            source.delete()
```

Comment:
not a priority, but is there a cleanup function that would delete this?

Reply:
Synced up offline. We currently don't have a cleanup function and we'll prolly create one in the future